docs/UserGuide.md

```diff
@@ -389,25 +389,98 @@ Find is case-insensitive, and the order in which the keywords are entered is irr
 
 <div markdown="block" class="alert alert-danger">
 
-:exclamation: **Caution:** You cannot undo the sort once you have applied it!
+:exclamation: **Caution:** 
+* You cannot undo the sort once you have applied it! 
+* Sort will not affect the list permanently. Trackermon rearranges the show list one time after a successful sort command! 
```

Comment:
I think we can use `once` instead of `one time` here. one time sounds a bit wrong to me haha.

Reply:
I was thinking abt that using once too. But once can mean when which might cause confusion.

Reply:
Hmm, I kinda see your point. When I'm going through the UG again hopefully I can find a way to rephrase it better haha.